birdhouse/config/postgres/credentials.env.template

```diff
@@ -1,3 +1,3 @@
 POSTGRES_USER=${POSTGRES_PAVICS_USERNAME}
 POSTGRES_PASSWORD=${POSTGRES_PAVICS_PASSWORD}
-POSTGRES_DB=pavics
+POSTGRES_DB=postgres-pavics
```

Comment:
Should merge with `master`.  I already fixed this problem here 56274f3e5fbdd9b4eb71bfd3acd3039dc965de7d.  This commit is not required anymore.